engine/src/flutter/shell/platform/darwin/macos/framework/Source/FlutterSurface.mm

```diff
@@ -15,6 +15,7 @@ @interface FlutterSurface () {
   id<MTLTexture> _texture;
   // Used for testing.
   BOOL _isInUseOverride;
+  MTLPixelFormat format;
```

Comment:
This doesn't seem to be used.

Reply:
(should have had an underscore prefix too)